### PR TITLE
Add native window handle to platform view

### PR DESCRIPTION
**Pull request by bwikbs**
* This is a preparation to independently handle something(e.g. imf) in platform view.

### CODE REVIEW

File: shell/platform/tizen/public/flutter_platform_view.h [hunk 1]
```diff
@@ -16,8 +16,9 @@ using ByteMessage = std::vector<uint8_t>;
 
 class PlatformView {
  public:
-  PlatformView(flutter::PluginRegistrar* registrar, int viewId)
-      : registrar_(registrar),
+  PlatformView(flutter::PluginRegistrar* registrar, int viewId, void* win)
+      : win_(win),
```
Comment (swift-kim) [inline shell/platform/tizen/public/flutter_platform_view.h:20]: Against the naming rule.

> Do not abbreviate by deleting letters within a word.

I recommend you to read the [style guide](https://google.github.io/styleguide/cppguide.html) from end to end, or at least the "Background" and "Naming" sections.

Comment (bwikbs) [reply shell/platform/tizen/public/flutter_platform_view.h]: It's fixed now!

File: shell/platform/tizen/tizen_renderer_evas_gl.cc [hunk 2]
```diff
@@ -569,6 +569,10 @@ uintptr_t TizenRendererEvasGL::GetWindowId() {
       ecore_evas_ecore_evas_get(evas_object_evas_get(evas_window_)));
 }
 
+void* TizenRendererEvasGL::GetWindowHandle() {
+  return nullptr;
```
Comment (swift-kim) [inline shell/platform/tizen/tizen_renderer_evas_gl.cc:573]: Is this intentional or just unimplemented?

**Comment by bbrto21** [comment]
> * This is a preparation to independently handle something(e.g. imf) in platform view.

I've been waiting for this for a long time. Thank you so much.

File: shell/platform/tizen/tizen_renderer_evas_gl.cc (hunk 2 above)
Comment (bwikbs) [reply shell/platform/tizen/tizen_renderer_evas_gl.cc:573]: It's half and half :smile: 
Evasgl version is not implemented yet.. so  This is for preventing tricky issues.

**Review by bbrto21 — APPROVED** [review]
LGTM

**Pull request merged**